src/components/Wizard/NewEditImportPane/ExperimentImportMessageForm.tsx

```diff
@@ -0,0 +1,365 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useEffect, useMemo, useState } from "react";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import Form from "react-bootstrap/Form";
+import {
+  useForm,
+  useWatch,
+  FormProvider,
+  SubmitHandler,
+  Controller,
+} from "react-hook-form";
+import { types, typeGuards } from "@mozilla/nimbus-shared";
+import Select from "react-select";
+
+import FormRow from "../FormRow";
+import deserialize from "../deserializers";
+import { useToastsContext } from "../../../hooks/useToasts";
+import { ImportMessageFormProps } from "./propTypes";
+
+type NimbusExperiment = types.experiments.NimbusExperiment;
+
+const EXPERIMENTS_API_URL =
+  "https://experimenter.services.mozilla.com/api/v6/experiments/";
+const EXPERIMENT_CACHE_KEY = "experiments";
+
+interface ExperimentOption {
+  value: string;
+  label: string;
+}
+
+/**
+ * Experimenter Feature IDs that we should filter by.
+ */
```

Comment:
Might be nice as an enhancement to add "filter by Live/Preview/Completed" to the form, since there's potentially a LOT of experiment recipes to search through

Reply:
We can't do that -- that information is not provided by the Nimbus API.

You can however search the returned results, which should be sufficient for finding your experiments.